doc/eventhandler/tokenhandler.rst

```diff
@@ -169,6 +169,16 @@ Using the action ``set failcounter`` you can reset the fail counter by
 setting it to 0 or also "block" the token by setting the fail counter to what
 ever value the "max_fail" is, e.g. 10. Only integer values are allowed.
 
+change failcounter
+..................
+
+Using the action ``change failcounter`` you can increase or decrease the failcounter.
+Positive and negative integer values are allowed.
```

Comment:
I think they are required. Using negative values will decrease the fail counter, using a positive value will increase the fail counter.

Reply:
ok